configuration/addresshierarchy/addressConfiguration.xml

```diff
@@ -0,0 +1,35 @@
+<addressConfiguration>
+  <wipe>true</wipe>
+  <addressComponents>
+    <addressComponent>
+      <field>COUNTRY</field>
+      <nameMapping>Country</nameMapping>
+      <sizeMapping>40</sizeMapping>
+      <elementDefault>addresshierarchy.Bahrain</elementDefault>
```

Comment:
This suggests that the default element is coded, but
1. For now we won't support i18n.
2. The CSV references `"Bahrain"` and not `"addresshierarchy.Bahrain"`.

Let's change this to 
```xml
<elementDefault>Bahrain</elementDefault>
```

Reply:
ok, i thought the first part is for filename , thanks for the correction :)